tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.html

```diff
@@ -314,9 +314,12 @@
       },
       _updateDownloadLink() {
         const svgStr = this.$$('tf-line-chart-data-loader').exportAsSvgString();
-        // https://github.com/tensorflow/tensorboard/pull/1446#issuecomment-436915402
-        // Firefox truncates href to some length for some reason. Using base64
-        // is okay.
+        // The SVG code string may include hash characters, such as an
+        // attribute `clipPath="url(#foo)"`. In such a case, Firefox
```

Comment:
Not just Firefox.  Evidently interpreting the `#` as identifying the fragment is what the spec requires:
https://lists.w3.org/Archives/Public/uri/2013Jul/0001.html

Chrome not doing the same thing is apparently a 7-year old bug that they finally pushed out a fix for in M72 in January: https://bugs.chromium.org/p/chromium/issues/detail?id=123004

Using base64 should still be fine, though I believe we could also just URL-encode the data (in which case `#` becomes `%23`) which has perhaps the slight advantage of preserving some human readability for formats like SVG that are at least quasi-readable.

Reply:
Ah, force of habit—I wrote 72 but actually tested on Chrome 67. Indeed,
in Chrome 72 the behavior is the same as in Firefox. Will fix; thanks!